backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -551,4 +564,30 @@ private static void checkSystemEndpoint(EndpointType endpointType) {
     private boolean isEndpointTypeAllowed(EndpointType endpointType) {
         return !featureFlipper.isEmailsOnlyMode() || endpointType.isSystemEndpointType;
     }
+
+    /**
+     * Removes the secrets from the endpoint's properties when returning them
+     * to the client.
+     * @param endpoint the endpoint to redact the secrets from.
+     */
+    @Deprecated(forRemoval = true)
+    protected void redactSecretsForEndpoint(final Endpoint endpoint) {
```

Comment:
It should be applyed only to user without write access, to keep existing behavior for org admins

Reply:
Why, though? We are not going to allow even org admins to see the secrets, no?

Reply:
Yes for Org admin on V2, but I understood we would like to keep the existing behavior on current version ?